### PR TITLE
Stop setting `--experimental_remote_download_regex` by default with command-line API

### PR DESCRIPTION
**Pull request by brentleyjones**
Also fixes the regex used to match what we changed `build_build.sh` to use.

This is to optimize bandwidth usage of cache warming calls.

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -316,6 +316,23 @@ For example, this will build all targets the same way that Xcode does:
 bazel run //:xcodeproj -- --generator_output_groups=all_targets build
 ```
 
+### `--download_intermediates`
```
Comment (brentleyjones) [inline docs/usage.md:319]: I'm not set on this name, if something else sounds better.

Comment (thiagohmcruz) [reply docs/usage.md:319]: Seems like a reasonable name to me

**Review by thiagohmcruz — APPROVED** [review]

**Review by mattrobmattrob — APPROVED** [review]

**Comment by mattrobmattrob** [comment]
Have any sense for the impact on a larger build's cache download before vs. after this change? I don't have a sense for how much these intermediates add in terms of download for a ~fully cached build.

**Comment by brentleyjones** [comment]
I don't. But the number of files (e.g. index stores) can be slow to write regardless. Just trying to make cache warming as efficient as possible.

**Pull request merged**